control-plane/tests/test_service_persistence_ingest_case_lifecycle.py

```
@@ -2663,15 +2663,141 @@ def test_service_alert_detail_excludes_case_only_sibling_evidence(self) -> None:
         }
 
         self.assertNotIn(sibling_evidence.evidence_id, linked_evidence_ids)
-        self.assertNotIn(sibling_evidence.evidence_id, detail.lineage["evidence_ids"])
+
+    def test_service_detail_surfaces_link_only_external_ticket_reference_on_alert_and_case(
+        self,
+    ) -> None:
+        store, service, promoted_case, _, _ = self._build_phase19_in_scope_case()
+
+        service.persist_record(
+            replace(
```

Ruff (0.15.10)

[error] Undefined name `replace`

(F821)